packages/next/server/next-server.ts

```diff
@@ -102,6 +97,7 @@ import ResponseCache, {
 } from './response-cache'
 import { NextConfigComplete } from './config-shared'
 import { parseNextUrl } from '../shared/lib/router/utils/parse-next-url'
+import Observable from 'zen-observable'
```

Comment:
```suggestion
import Observable from 'next/dist/compiled/zen-observable'
```
Can we `ncc` this to reduce the size a bit? https://packagephobia.com/result?p=zen-observable

Reply:
Will give it a try